gammapy/irf/tests/test_energy_dispersion.py

```diff
@@ -1,13 +1,41 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from numpy.testing import assert_allclose
+from numpy.testing import assert_allclose, assert_equal
 from astropy.tests.helper import pytest
-from ...irf import EnergyDispersion
-
+from ...irf import EnergyDispersion, EnergyDispersion2D
+from ...datasets import load_edisp2D_fits_table
 
 @pytest.mark.xfail
 def test_EnergyDispersion():
     edisp = EnergyDispersion()
     pdf = edisp(3, 4)
     assert_allclose(pdf, 42)
+
```

Comment:
You have to skip this test if scipy is not installed:
https://travis-ci.org/gammapy/gammapy/jobs/77671811#L1037
http://astropy.readthedocs.org/en/latest/development/testguide.html#tests-requiring-optional-dependencies